solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttp2SolrClient.java

```diff
@@ -94,35 +97,63 @@
  *
  * @since solr 8.0
  */
-public class LBHttp2SolrClient<C extends HttpSolrClientBase> extends LBSolrClient {
+public class LBHttp2SolrClient<B extends HttpSolrClientBuilderBase<?, ?>> extends LBSolrClient {
```

Comment:
an aside -- I wonder if the use of generics here is more annoying than helpful.  We aren't compelled to use generics.

Reply:
I worry too I have gone down the wrong road with this.  On the other hand, I do not want consider this with this PR.

Reply:
in another PR then.